shared/src/types/base/category.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from 'zod'
+import { BaseModel } from './common'
+
+export const Category = BaseModel.extend({
+  catname: z.string(),
+  parentId: z.string(),
```

Comment:
Is the parentId supposed to be a number?

Reply:
yup thanks for spotting!